modules/cli/pkg/runs/runsGet.go

```diff
@@ -89,27 +89,23 @@ func GetRuns(
 		chosenFormatter, err = validateOutputFormatFlagValue(outputFormatString, validFormatters)
 		if err == nil {
 			var runJson []galasaapi.Run
-			runJson, err = GetRunsFromRestApi(runName, requestorParameter, resultParameter, fromAge, toAge, shouldGetActive, timeService, commsClient, group)
-			if err == nil {
-				// Some formatters need extra fields filled-in so they can be displayed.
-				if chosenFormatter.IsNeedingMethodDetails() {
-					log.Println("This type of formatter needs extra detail about each run to display")
-					runJson, err = GetRunDetailsFromRasSearchRuns(runJson, commsClient)
```

Comment:
Is `GetRunDetailsFromRasSearchRuns` still being used elsewhere? If not, might be worth removing it.